core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -120,6 +120,19 @@ pre {
   border: none;
 }
 
+.description-input {
+  overflow: hidden !important;
```

Comment:
yes, in my debug there is other style will take precedence here. anyway, new version i do not use input and remove "!important"